sdk/storage/microsoft-azure-storage-blob/pom.xml

```diff
@@ -73,22 +73,6 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-core-test</artifactId>
-      <version>1.23.0</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-storage-common</artifactId>
-      <version>12.24.2</version> <!-- {x-version-update;com.azure:azure-storage-common;dependency} -->
-      <classifier>tests</classifier>
-      <type>test-jar</type>
-      <scope>test</scope>
-    </dependency>
```

Comment:
Why did we have these dependencies? Looks like it was not being used at all except for `LiveOnly` import but don't see that used in code.